src/index.ts

```diff
@@ -87,36 +80,15 @@ async function doBuildFromScratch(cli: BuildahCli, newImage: string, workspace:
 function getInputList(name: string): string[] {
     const items = core.getInput(name);
     if (!items) {
-      return [];
+        return [];
     }
     return items
-      .split(/\r?\n/)
-      .filter(x => x)
-      .reduce<string[]>(
-          (acc, line) => acc.concat(line).map(pat => pat.trim()),
-          []
+        .split(/\r?\n/)
+        .filter((x) => x)
```

Comment:
We don't have to change this right now but this filter call does nothing and should be removed